src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/AvoidConstArraysTests.cs

```diff
@@ -0,0 +1,338 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.AvoidConstArraysAnalyzer,
+    Microsoft.NetCore.Analyzers.Runtime.AvoidConstArraysFixer>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.AvoidConstArraysAnalyzer,
+    Microsoft.NetCore.Analyzers.Runtime.AvoidConstArraysFixer>;
+
+namespace Microsoft.NetCore.Analyzers.Runtime.UnitTests
+{
+    public class AvoidConstArraysTests
```

Comment:
Here are a couple of extra tests you could add:

- specific trivias (comment, maybe line break) to see if the fixer is doing what we would expect.
- array using constants defined in variable (local and fields)
- test where values are `static readonly` (so not const)

Reply:
@Evangelink Done! I added a test for your first point. For your third point I already tested non-literal arrays in the NoDiagnostic method. For the third, the only values we look at are local -- those included in local invocations or arguments -- so I added no test for that. I'll make sure that my tests pass.

Reply:
@Evangelink all of the tests I added according to your feedback passed.